src/ui/styles/app.css

```diff
@@ -0,0 +1,2 @@
+@import 'todomvc-app-css/index.css';
```

Comment:
Can you concat the two files into `app.css` (in the `ember-cli-build.js`) instead of adding an additional requests via `@import`?